view/bindings/bindings.js

```diff
@@ -236,8 +236,15 @@ steal("can/util", "can/view/mustache", "can/control", function (can) {
 			if (!this.element) {
 				return;
 			}
+			var el = this.element[0];
+			
 			// Set the value of the attribute passed in to reflect what the user typed
-			this.options.value(this.element[0].value);
+			var newVal = this.options.value(el.value);
```

Comment:
I don't think can.compute necessarily always returns its current value when being set.  I would change this to:

```
this.options.value(el.value)
var newVal = this.options.value();
```